server.py

```diff
@@ -964,12 +964,54 @@ def get(self, command):
         return "Asset switched"
 
 
+class AssetContent(Resource):
+    method_decorators = [api_response, auth_basic]
+
+    @swagger.doc({
+        'parameters': [
+            {
+                'name': 'asset_id',
+                'type': 'string',
+                'in': 'path',
+                'description': 'id of an asset'
+            }
+        ],
+        'responses': {
+            '200': {
+                'description': 'The binary content of the asset'
+            }
+        }
+    })
+    def get(self, asset_id):
+        with db.conn(settings['database']) as conn:
+            asset = assets_helper.read(conn, asset_id)
+
+        if isinstance(asset, list):
+            return {'error': 'Invalid asset ID provided'}, 404
```

Comment:
`raise Exception('Invalid asset ID provided')`

because we have [this handler](https://github.com/Screenly/screenly-ose/blob/master/server.py#L344)